### PR TITLE
[WIP] Created ES.properties

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by JonathanBrouwer** [comment]
Mmmm, 2 PR's for the same language lol

File: src/main/resources/Messages/ES.properties [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+uptimeMessage=@1Uptime: @2%Time
+uuidUsage=@3/usage @2<Player>
+uuidMessage=@1The uuid of @2%Name @1is @2%Uuid@1.
+vanishSelf=@1Vanish mode is now @2%Status@1.
+vanishSelfT=@1Vanish mode is now @2%Status @1for @2%Time@1.
+vanishOthersSelfMessage=@1Your vanish mode from @2%Player @1is now @2%Status@1.
+vanishOthersOtherMessage=@1Your vanish mode is now @2%Status@1.
+vanishOthersSelfMessageT=@1Your vanish mode from @2%Player @1is now @2%Status @1for @2%Time@1.
+vanishOthersOtherMessageT=@1Your vanish mode is now @2%Status @1for @2%Time@1.
+vanishTarget=@1You are now vanished.# To change this license header, choose License Headers in Project Properties.
```
Comment (JonathanBrouwer) [inline src/main/resources/Messages/ES.properties:475]: Still weird note

Comment (ghost) [inline src/main/resources/Messages/ES.properties:475]: i will work in that

**Comment by JonathanBrouwer** [comment]
Closed because of inactivity.
If you wish to continue comment on this issue.

**Pull request closed without merging**